README.md

```diff
@@ -0,0 +1,103 @@
+# Flask Project Boilerplate
+
+This repository is a boilerplate Flask project for quickly getting started.
+
+## Getting started
+
+Steps:
+
+1. Clone/pull/download this repository
+2. Create a virtualenv with `python3 -m venv env_py37` and install dependencies with `pip install -r requirements.txt`
+4. Configure your `.env` variables
```

Comment:
Add here note that developer should create `.env` file first. Check my previous comment for `.env`.